VideoLocker/src/main/java/org/edx/mobile/view/CourseCombinedInfoFragment.java

```diff
@@ -341,19 +335,31 @@ private void populateAnnouncements(List<AnnouncementsModel> announcementsList) {
         if(announcementsList !=null && announcementsList.size()>0) {
             hideEmptyAnnouncementMessage();
 
-            final String divider = "<div style='height:2px; width:100%; background-color: #E2E3E5'></div>";
-
-            StringBuffer buff = new StringBuffer();
-
-            // add meta viewport
-            buff.append("<meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">");
-
-            for (AnnouncementsModel m : announcementsList) {
-                String entry = String.format("%s <br/><br/> %s <br/> %s <br/><br/>", m.getDate(), divider, m.getContent());
-                buff.append(entry);
+            try{
+                String cssFileContent = FileUtil.loadTextFileFromAssets(getActivity(), "css/handouts-announcements.css");
+                StringBuffer buff = new StringBuffer();
+                // add meta viewport
+                buff.append("<meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">");
+                // add css file
+                buff.append("<head>");
+                buff.append("<style>");
+                buff.append(cssFileContent);
```

Comment:
hmm, I discussed this with @1zaman as well. But after reading this line "Better to just not print anything if loading this fails." [here](https://github.com/edx/edx-app-android/pull/311#discussion_r36109899), we decided to go with the current solution. Also, we tried loading the html as it is without the CSS file, and it looked fine. So, I guess loading something is better than nothing :)